server/data/data-jpa/pom.xml

```diff
@@ -33,7 +33,7 @@
     <name>Apache James :: Server :: Data :: JPA Persistence</name>
 
     <properties>
-        <target.jdk>11</target.jdk>
+        <target.jdk>21</target.jdk>
```

Comment:
Is this even needed to get this property defined here?

We likely should just inherit this from parent project...

Reply:
Yes indeed, I'll try to inherit this